mmedit/models/components/layers/mean_shift.py

```diff
@@ -0,0 +1,27 @@
+import torch
+import torch.nn as nn
+
+from mmedit.models.registry import COMPONENTS
+
+
+@COMPONENTS.register_module()
+class MeanShift(nn.Conv2d):
+    """Mean shift.
+
+    Args:
+        pixel_range (float): Pixel range of image.
+        rgb_mean (Tuple[float]): Image mean in RGB orders.
+        rgb_std (Tuple[float]): Image std in RGB orders.
+        sign (int): Sign of bias. Default -1.
+    """
+
+    def __init__(self, pixel_range, rgb_mean, rgb_std, sign=-1):
+        super(MeanShift, self).__init__(3, 3, kernel_size=1)
+        std = torch.Tensor(rgb_std)
```

Comment:
What's the device of this tensor

Reply:
CPU.
It is not important, only used in init.

Reply:
No this should be taken care, becuase it might be run on gpu

Reply:
How about `del std` after __init__?

Reply:
My understanding was wrong. No need to change :)